js/classes/inventory.js

```diff
@@ -185,35 +186,43 @@ class Inventory {
         return this.currentFood;
     }
 
-
-
-    /**
-     * @param {number} type The type of the rune to add to the inventory.
-     * @param {number} amount The amount of rune to add to the inventory.
-     */
-    addRune(type, amount) {
-
-    }
-
-    /**
-     * @returns {Array} An array of all the rune in the inventory.
-     */
-    getAllRunes() {
-        return this.runeBag;
-    }
-
-    /**
-     * @param {number} type The type of the rune to get.
-     * @returns {RuneItem} The rune with the given name.
-     */
-    getRune(type) {
-        for (let i = 0; i < this.runeBag.length; i++) {
-            let rune = this.runeBag[i];
-            if (rune.type == type) {
-                return this.runeBag[i];
-            }
-        }
-    }
+    spendRunes(runes) {
+        this.runes -= runes;
+        HUD.runeCounter.setCount(this.runes);
+    };
```

Comment:
I would probably just do an `incrementRunes` function or something like that and just pass in a negative value instead of having two mostly the same functions to handle this.

Reply:
let's do adjustRunes(amt)